modules/dnn/src/ocl4dnn/src/ocl4dnn_conv_spatial.cpp

```diff
@@ -1826,7 +1824,7 @@ bool OCL4DNNConvSpatial<Dtype>::loadCachedConfig()
         for (size_t i = 0; i < numConfigs; i++)
         {
             std::pair<std::string, std::string> entry(
-                    std::string("Intel(R) Corporation_") + default_kernel_config_intel[2 * i],
+                    ocl::Device::getDefault().vendorName() + "_" + default_kernel_config_intel[2 * i],
```

Comment:
These config numbers for Intel devices only.
`ocl::Device::getDefault()` can point to any OpenCL device, including AMD / NVIDIA.

Did you observe name changing in some Intel OpenCL runtime (platform / version)?

Reply:
I just thought "intel" is the default vendor here, but it looks not the case for other users. changed back to old string.